scripts/jenkins/init-node.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install nvm for this shell
+source ~/.nvm/nvm.sh
+
+# Use version of node.js found in .nvmrc
+nvm install
+
+npm install -g yarn
```

Comment:
Some environments, like Heroku, use `engines` to decide what version to use. I guess it won't happen in our situation but we should still specify the version somewhere and it doesn't hurt to do it in `engines` and read from there; if someone wanted to do sth on one of these engines-reading tools, they could use these settings.

As for humans, we could always add a check for an installed version and require it to be not older than whatever's in `engines.yarn`.